src/main/java/org/folio/consortia/service/UserTenantService.java

```diff
@@ -0,0 +1,21 @@
+package org.folio.consortia.service;
+
+import org.folio.pv.domain.dto.UserTenant;
+import org.folio.pv.domain.dto.UserTenantCollection;
+
+import java.util.UUID;
+
+public interface UserTenantService {
+
+  /**
+   * Get user tenant associations collection by query
+   *
+   * @param offset the offset
+   * @param limit  the limit
+   * @return the user tenant associations collection
+   */
+  UserTenantCollection get(UUID userId, String username, Integer offset, Integer limit);
```

Comment:
We should have separate service method to find by userId, and another - to find by username

Reply:
Ok